Packages/ClientRuntime/Tests/ClientRuntimeTests/PrimitiveTypeExtensionsTests/StringExtensionsTests.swift

```diff
@@ -96,4 +96,14 @@ class StringExtensionsTests: XCTestCase {
             XCTAssertEqual(string.substringBefore(":"), match)
         }
     }
+    
+    func testNilOrEmptyExtension() {
+        let emptyString: String? = ""
+        let nilString: String? = nil
+        let stringWithValue: String? = "something"
+        
+        XCTAssertFalse(stringWithValue.isNilOrEmpty)
+        XCTAssertTrue(nilString.isNilOrEmpty)
+        XCTAssertTrue(emptyString.isNilOrEmpty)
+    }
```

Comment:
It would be better to split this out in to 3 different functions to reduce the scope of each case.

Reply:
removing as no longer needed